lib/consts/index.ts

```diff
@@ -1145,3 +1146,391 @@ export const accordionInfo: AccordionInfoType[] = [
       "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam euismod orci sed tristique placerat. Fusce in ligula urna. Fusce eget nunc et libero accumsan rutrum quis nec lectus. Quisque luctus sem nibh, quis ornare neque consectetur varius. Maecenas rhoncus consectetur rutrum.",
   },
 ];
+
+export const faqData: FaqDataType[] = [
+  {
+    id: 1,
+    buttonText: "General",
+    buttonCategory: "generalquestion",
```

Comment:
@oscarcorellan The value generalquestion should be on camelCase
```suggestion
    buttonCategory: "generalQuestion",
```